src/main/resources/application.yaml

```diff
@@ -181,6 +181,8 @@ darts:
     retention-period:
       inbound:
         arm-minimum: 24
+        unstructured-minimum:
+          hours: 24
```

Comment:
Same comment as above regarding the format here.

Reply:
Its consistent with the naming convention for the one below, lines 186-188, which I think is a better naming convention. If you they should all be consistent which way is best to go for?